modules/postgres/testcontainers/postgres/__init__.py

```diff
@@ -79,7 +79,7 @@ def get_connection_url(self, host: Optional[str] = None, driver: Optional[str] =
         driver. The optional driver argument to :code:`get_connection_url` overwrites the constructor
         set value. Pass :code:`driver=None` to get URLs without a driver.
         """
-        driver_str = self.driver if driver is _UNSET else f"+{driver}"
+        driver_str = "" if driver is None else self.driver if driver is _UNSET else f"+{driver}"
```

Comment:
I would rather `self.driver` did not contain the leading `+`, then this could be simpler. But I imagine that could be a breaking change if people are relying on the internals of this class

Reply:
AFAICS, `driver` does not contain a `+`, it is added and stored in `driver_str`. So the only change this would add is that if you pass in `None`, you get an empty driver string.

I think that is behavior more aligned with what people would expect, so I think this is good2go

Reply:
Agreed. I think we can introduce a Breaking change at a later stage, as part of some larger changes to inheritance for DB containers. This only looks good for now, and I think people would expect that if you pass `None`, the driver should be empty.